geomesa-accumulo/geomesa-accumulo-datastore/src/main/scala/org/locationtech/geomesa/accumulo/data/AccumuloFeatureReader.scala

```diff
@@ -98,6 +100,7 @@ class AccumuloFeatureReaderWithStats(query: Query,
       timings.time("next") + timings.time("hasNext"),
       count
     )
+    Monitoring.log(stat)
```

Comment:
I was thinking about that...  Here's where I ended up...

It seems that one may want to vary 1) how stats are written, 2) where stats are written, and 3) extend UsageStat in their own project.

With those three ideas, it seems like any stats writers would need to have an 'accepts' / 'canProcess' method to see if it should be wired up.  If it happens at the stat object level, there's lots of type checking.  Otherwise, the writer need to be wired up at start-up.

Generally, I was imagining a case where stats could be written to Accumulo, ElasticSearch, Graphite, or disk.  Each of those might require one or three different output formats (csv, tsv, json?).  That turned into a bit of a nightmare to think through...

At the same time, I realized that most of the other requirements (multiple sources, lazy evaluation, 'easy' to configure, log file rotation, etc) were all handled by turfing it to Scala Logging;).

If we stick with this kind of approach, we can write out json to disk, and then use external tooling to take it from there...